meta-facebook/yv35-cl/src/platform/plat_class.c

```diff
@@ -113,8 +167,8 @@ void init_platform_config()
 	i2c_msg = construct_i2c_message(I2C_BUS1, CPLD_ADDR, tx_len, data, rx_len);
 	if (!i2c_master_read(&i2c_msg, retry)) {
 		class_type = i2c_msg.data[0];
-		is_1ou_present = (class_type & 0x4 ? false : true);
-		is_2ou_present = (class_type & 0x8 ? false : true);
+		_1ou_status.present = (class_type & BIT(2) ? false : true);
+		_2ou_status.present = (class_type & BIT(3) ? false : true);
```

Comment:
Add Parenthesis just to be safe.

Reply:
We has fixed this. Please review it again, thanks.